.circleci/config.yml

```diff
@@ -21,11 +21,14 @@ jobs:
       - run:
           command: yarn build:glean
           name: generate glean files
+      - run: curl -Os https://uploader.codecov.io/v0.1.0_4653/linux/codecov
+      - run: chmod +x codecov
       - run:
           command: yarn test:coverage
           name: Run Jest tests
       - run:
-          command: yarn codecov
+          command: ./codecov -t ${CODECOV_TOKEN} --required -f coverage.xml
```

Comment:
We do not generate an xml file for the JS coverage; it is a directory. We may want to drop ` --required -f coverage.xml` from the command.

Alternatively, we could keep the steps in `javascript-tests` as is and modify the `yarn` command itself by replacing the following line:
https://github.com/mozilla/treeherder/blob/3c669cd02743717297fcecbe598de474db730ace/package.json#L123
with:
```json
    "codecov": "rm -rf coverage && curl -Os https://uploader.codecov.io/v0.1.0_4653/linux/codecov && chmod +x codecov && ./codecov -t ${CODECOV_TOKEN}",
```